src/test/scala/com.snowplowanalytics.sauna/observers/LocalObserverTest.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2015 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.sauna.observers
+
+import java.io.{File, InputStream, PrintWriter}
+import java.util.UUID
+
+import com.snowplowanalytics.sauna.loggers.MutedLogger
+import com.snowplowanalytics.sauna.processors.Processor
```

Comment:
You can set package as following:
```scala
package com.snowplowanalytics.sauna
package observers
```
Then you can could use more concise imports:
```scala
import loggers.MutedLogger
```